src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    isMarried,
+    sex,
+    partnerName,
+  } = person;
+  const partner = sex === 'f' ? 'husband' : 'wife';
```

Comment:
try to avoid magic values `const FIMALE = 'f'`
```suggestion
  const partner = sex === FIMALE ? 'husband' : 'wife';
```